content/configuration/policy_in_gocd.md

```diff
@@ -0,0 +1,67 @@
+---
+description: GoCD authorization via policy
+keywords: GoCD configuration, authorization, administration, users, roles, role-based security, security, security configuration, user permissions, GoCD administration, role-based access control, rbac
+title: Policy in GoCD
+---
+
+# Policy
+
+A policy is a set of permissions for GoCD entities. This helps restricts the entity access based on the user's role within GoCD. GoCD supports a restrictive model of access i.e. until access is granted, the entity can't be accessed by anyone (except admins).
```

Comment:
A policy is a set of permissions for GoCD entities such as environments, configuration repositories, etc. The policy defines the access of a GoCD entity for the users in a role.